lib/vsc/install/ci.py

```diff
@@ -201,10 +201,17 @@ def indent(line, level=1):
         'python2.7 -V',
     ]
 
+
     if vsc_ci_cfg[PIP_INSTALL_TOX]:
-        test_cmds.append('pip install --ignore-installed --user tox')
+        test_cmds.extend([
+            'pip install --user --upgrade pip',
```

Comment:
why not for pip3?

Reply:
Because I haven't run into problems yet with `pip3` on `jenkins10`... :)